build.gradle

```diff
@@ -66,6 +66,14 @@ repositories {
     maven { url "http://maven.cil.li/" }
     maven { url "http://maven.ic2.player.to/" }
     maven { url "http://tehnut.info/maven" }
+	ivy {
+        name 'Mekanism'
+        artifactPattern "http://addons-origin.cursecdn.com/files/${config.mekanism.cf}/[module]-[revision].[ext]"
```

Comment:
You should not need the full jar, nor to use curse.  API jars are available on Aidan's site or my maven @ http://maven.thiakil.com/Mekanism/